scripts/buf_lint_check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Maybe call file: ``proto_lint_check`` for parity with the Kotlin one (users care what language is being linted, not what tool is doing the linting).